jekyll/_cci2/env-vars.md

```diff
@@ -80,21 +101,68 @@ Login Succeeded
 Not all command-line programs take credentials
 in the same way that `docker` does.
 
-## Adding Global Environment Variables
+## Setting Context-Level Environment Variables
```

Comment:
I think Context-Level is awkward and not the best for translation either, so I would prefer Setting Environment Variables for a Context/ with a Context. But, and was reviewed with the context documentation previously, so it should stay as-is really.

Reply:
Yeah, I agree that the hyphenated combo is awkward. I do like using the word "level" since it's a common term in discussions of [scope](https://en.wikipedia.org/wiki/Scope_(computer_science)#Levels_of_scope).

What about "Setting Environment Variables at the Context Level"?

Reply:
We don't use level, we just say context. 'in a context' or 'on the contexts page'